sonar-java-plugin/src/main/java/org/sonar/plugins/java/JavaRulesDefinition.java

```diff
@@ -151,10 +160,16 @@ private static void addMetadata(NewRule rule, @Nullable RuleMetadata metadata) {
     addSecurityStandards(rule, metadata.securityStandards);
   }
 
-  private static void addSecurityStandards(NewRule rule, SecurityStandards securityStandards) {
-    for (String s : securityStandards.OWASP) {
+  private void addSecurityStandards(NewRule rule, SecurityStandards securityStandards) {
+    for (String s : securityStandards.OWASP_2017) {
       rule.addOwaspTop10(OwaspTop10.valueOf(s));
```

Comment:
I am surprised Next didn't report usage of this deprecated API.

Reply:
since you didn't modify the line, it might have excluded the issue from the report.